apps/admin/src/components/create/EventFormRow.tsx

```diff
@@ -0,0 +1,32 @@
+import type { ReactNode } from "react";
+import type { FieldErrors, FieldValues } from "react-hook-form";
+
+interface EventFormRowProps<T extends FieldValues> {
+  label: string;
+  name: keyof T;
+  children: ReactNode;
+  errors: FieldErrors<T>;
+}
```

Comment:
`errors` 전체를 받는게 맞을까요? 실제 에러 메시지가 변경되지 않아도 불필요한 리렌더링이 발생할 것 같아요.

```suggestion
interface EventFormRowProps<T extends FieldValues, N extends keyof T> {
  label: string;
  name: N;
  children: ReactNode;
  error: FieldErrors<T>[N] | undefined;
}
```